libs/api/domains/national-registry-x-road/src/dto/nationalRegistryChildGuardianshipInput.ts

```diff
@@ -0,0 +1,7 @@
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType()
```

Comment:
We should use the api domain name as prefix for the type to avoid naming collisions in the GQL schema.
```suggestion
@InputType('NationalRegistryXRoadChildGuardianshipInput')
```